Revit_Core_Engine/Compute/CreateTransformed.cs

```diff
@@ -0,0 +1,94 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using BH.oM.Base.Attributes;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Return transformed bounding box that is created from transformed corner points.")]
+        [Input("bbox", "Bounding box to transform.")]
+        [Input("transform", "Transformation of the bounding box.")]
+        [Output("bbox", "Transformed bounding box.")]
+        public static BoundingBoxXYZ CreateTransformed(this BoundingBoxXYZ bbox, Transform transform)
+        {
+            if (bbox == null)
+                return null;
+
+            BoundingBoxXYZ newBBox = new BoundingBoxXYZ() { Min = bbox.Min, Max = bbox.Max };
```

Comment:
What about `bbox.Transform`? Shouldn't you do something like below?
```suggestion
            BoundingBoxXYZ newBBox = new BoundingBoxXYZ() { Min = bbox.Transform.OfPoint(bbox.Min), Max = bbox.Transform.OfPoint(bbox.Max) };
```

...with necessary null checks on transform ofc.